database/models.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
```

Comment:
Suggestion (non-blocking): since these two imports are not used, we can remove these when refactoring code.